app_flutter/lib/status_grid.dart

```diff
@@ -29,11 +36,28 @@ class StatusGridContainer extends StatelessWidget {
         final List<CommitStatus> statuses = buildState.statuses.data;
 
         if (buildState.hasError) {
-          print('FlutterBuildState has an error');
           print('isTreeBuilding: ${buildState.isTreeBuilding.error}');
           print('statuses: ${buildState.statuses.error}');
 
-          // TODO(chillers): Display the error
+          SchedulerBinding.instance.endOfFrame.then((_) {
```

Comment:
I don't think you should trigger a side-effect like showing a snackbar as a side effect of a build method. An alternative way to structure this would be for the buildStatus to include a change notifier that you can subscribe to in initState/dispose. Then trigger the snackbar there, which removes the need for the post frame callback

Reply:
Would this require switching the `hasErrors` member in `FlutterBuildState` to its own `ChangeNotifier`? Or should I make it a `Stream` that can be subscribed to?

Reply:
I think ChangeNotifiers are generally preferable to Streams, but both work. Streams don't have a "current" value, which makes timing issues easier to hit

Reply:
Added!